### PR TITLE
Add some new WAM docs

### PR DESCRIPTION
**Pull request by vtbassmatt**
Adds a few notes about WAM and its integration into the Windows user account.

### CODE REVIEW

**Comment by vtbassmatt** [comment]
@mjcheetham I know this will require changing an aka.ms link. As we discussed, I think this filename has better browsability.

File: docs/windows-broker.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+This is due to a configuration outside of our control.
+We expect this experience to improve over time and a "personal account" option to be presented in the future.
+
+![Initial dialog to choose an existing or different account](img/get-signed-in.png)
+
+If you've connected your MSA to Windows or signed-in to other Microsoft applications such as Office, then you may see this account listed in the authentication prompts when using GCM.
+For any connected MSA, you can control whether or not the account is available to other Microsoft applications in **Settings**, **Accounts**, **Emails & accounts**:
+
+![Allow all Microsoft apps to access your identity](img/all-microsoft.png)
+![Microsoft apps must ask to access your identity](img/apps-must-ask.png)
```
Comment (mjcheetham) [inline docs/windows-broker.md:63]: Stick these in a table to get them side-by-side (with appropriate captions) so the layout is a little better (no more GIANT images)?

**Review by mjcheetham — APPROVED** [review]
Looks good, apart from maybe laying out some of the GIANT images better (see comment for one of them(.

File: docs/windows-broker.md (hunk 1 above)
Comment (vtbassmatt) [reply docs/windows-broker.md]: I added a newline between them so they stack instead. Thanks!

**Pull request merged**